domains/subhan.json

```
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "MS1034",
+        "email": "muhammadsubhan5701@gmail.com"
+    },
+    "record": {
+        "CNAME": "https://muhammad-subhan.vercel.app"
```

CNAME cannot contain protocol
```suggestion
        "CNAME": "muhammad-subhan.vercel.app"
```